src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -112,9 +112,10 @@ def _upload_fuzz_strategy_weights(client):
   for row in data:
     curr_strategy = data_types.FuzzStrategyProbability()
     curr_strategy.strategy_name = str(row['strategy'])
-    curr_strategy.strategy_count = int(row['strategy_count'])
     curr_strategy.strategy_probability = float(row['bandit_weight'])
     strategy_data.append(curr_strategy)
+  ndb.delete_multi(
+      data_types.FuzzStrategyProbability.query().fetch(keys_only=True))
```

Comment:
Sorry to make you update this again, but could you try this with a list comprehension along the lines of:

[entity.key for entity in ndb_utils.get_all_for_query(...)]

If there are issues with that for whatever reason it's fine as-is. It looks like we do this in a few other places throughout the code without using get_all_for_query.